test/vanilla/Expected/AcceptanceTests/Validation/validation/models/_models.py

```diff
@@ -114,8 +114,8 @@ class Product(msrest.serialization.Model):
     :vartype const_int: int
     :ivar const_string: Required. Constant string. Default value: "constant".
     :vartype const_string: str
-    :ivar const_string_as_enum: Constant string as Enum. Default value: "constant_string_as_enum".
-    :vartype const_string_as_enum: str
+    :param const_string_as_enum: Constant string as Enum.
```

Comment:
[swagger](https://github.com/Azure/autorest.testserver/blob/main/swagger/validation.json#L206)

Not required, no default value